README.md

```diff
@@ -51,8 +49,24 @@ The implementation of `remove-unit` allows the operator to remove more than one
 
 Supported [relations](https://juju.is/docs/olm/relations):
 
-There are currently no supported relations.
+#### New `mongodb-client` interface:
+
+Relations to new applications are supported via the `mongodb-client` interface. To create a relation: 
+
+```shell
+juju relate mongodb application
+```
```

Comment:
Since this lives with the legacy relation, do you need to specify the name of the interface?
```
juju relate mongodb:mongodb-client application
```
or
```
juju relate mongodb:mongodb application
```

Reply:
when you do juju relate it scans the two interface lists and looks for a match. In this case mongodb supports both a 'mongodb' interface via its 'obsolete' endpoint. Graylog also declares a 'mongodb' interface, so when you do juju deploy, juju sees a matching interface name and relates mongodb:obsolete --> graylog: